src/Razor/src/Microsoft.AspNetCore.Razor.LanguageServer/CodeActions/CSharp/DefaultCSharpCodeActionProvider.cs

```diff
@@ -59,13 +59,18 @@ internal class DefaultCSharpCodeActionProvider : CSharpCodeActionProvider
                 return EmptyResult;
             }
 
-            var results = codeActions.Where(codeAction =>
-                StringMatchCodeActions.Contains(codeAction.Title) ||
-                RegexMatchCodeActions.Any(pattern => pattern.Match(codeAction.Title).Success)
-            );
+            var results = new List<RazorCodeAction>();
+
+            foreach (var codeAction in codeActions)
+            {
+                if (SupportedDefaultCodeActionNames.Contains(codeAction.Name))
+                {
+                    results.Add(codeAction);
+                }
+            }
 
             var wrappedResults = results.Select(c => c.WrapResolvableCSharpCodeAction(context)).ToList();
```

Comment:
Why loop over this again instead of calling WrapResolvableCSharpCodeAction inside the foreach loop above?

Reply:
Ah good call, removed.